src/views/Stake/Stake.tsx

```diff
@@ -163,7 +164,7 @@ function Stake() {
       return dispatch(error(t`You cannot stake more than your OHM balance.`));
     }
 
-    if (checked === false && action === "unstake" && gweiValue.gt(ethers.utils.parseUnits(sohmBalance, "gwei"))) {
+    if (confirmation === false && action === "unstake" && gweiValue.gt(ethers.utils.parseUnits(sohmBalance, "gwei"))) {
       return dispatch(
         error(
           t`You do not have enough sOHM to complete this transaction.  To unstake from gOHM, please check the box.`,
```

Comment:
I'm not in love with this feedback: 
![image](https://user-images.githubusercontent.com/80423742/147700751-85602c87-b39f-46e3-a9df-f0eef873b3de.png)
should it be "To unstake from gOHM, please toggle the sohm-gohm switch."